terraform/monitoring/provider.tf

```diff
@@ -4,4 +4,15 @@ provider "cloudfoundry" {
   password = var.password
 }
 
+terraform {
+  required_version = ">= 0.13.4"
+
+  required_providers {
+    cloudfoundry = {
+      source  = "cloudfoundry-community/cloudfoundry"
+      version = "0.12.6"
+    }
+  }
+}
+
```

Comment:
extra new line? also line 58 in modules.tf file

Reply:
ok .. next time it is changed I will pickit up